Chummer/Backend/Equipment/Drugs.cs

```diff
@@ -1264,13 +1264,16 @@ public async Task<string> GenerateDescriptionAsync(int intLevel = -1, bool blnEf
                         sbdDescription.AppendLine(await _objCharacter.TranslateExtraAsync(strInfo, strLanguage, token: token).ConfigureAwait(false));
 
                     if (Category == "Custom Drug" || Duration != 0)
-                        sbdDescription.Append(await LanguageManager.GetStringAsync("Label_Duration", strLanguage, token: token).ConfigureAwait(false))
-                                      .AppendLine(DisplayDuration);
+                        sbdDescription.Append(await LanguageManager.GetStringAsync("Label_Duration", token: token).ConfigureAwait(false))
+                                      .Append(strSpace)
+                                      .Append("10 ⨯ ")
+                                      .Append((Duration + 1).ToString(GlobalSettings.CultureInfo))
+                                      .Append(await LanguageManager.GetStringAsync("String_D6", token: token).ConfigureAwait(false)).Append(strSpace)
+                                      .AppendLine(await LanguageManager.GetStringAsync("String_Minutes", token: token).ConfigureAwait(false));
```

Comment:
I would prefer it if the DisplayDuration property itself (and/or a newly-added async getter version to support `LanguageManager.GetStringAsync`) were fixed instead of only applying the fix to the specific part where it is used to generate the drug's description. There are probably other places where getting the displayed version of a drug's duration might be useful beyond just the description block in the UI that's meant to show all information.